web/src/components/RegisteredForTaxesModal.test.tsx

```diff
@@ -10,8 +9,13 @@ import {
   triggerQueueUpdate,
   WithStatefulUserData,
 } from "@/test/mock/withStatefulUserData";
-import { LegalStructures, LookupOwnershipTypeById, UserData } from "@businessnjgovnavigator/shared";
-import { generateMunicipality } from "@businessnjgovnavigator/shared/test";
+import {
+  generateProfileData,
+  LegalStructures,
+  LookupOwnershipTypeById,
+  UserData,
+} from "@businessnjgovnavigator/shared";
+import { generateMunicipality, generateUserData } from "@businessnjgovnavigator/shared/test";
```

Comment:
So I was debating about when and whether to divide up my imports between @shared and shared/test. I had no strong feelings about this so things are a bit inconsistent in some places based on what the file was already doing before I added things. I think this is fine but if someone heavily disagrees lmk